features/pageobjects/addRemoveElementPage.js

```diff
@@ -0,0 +1,34 @@
+const { $ } = require("@wdio/globals");
+const Page = require("./page");
+
+class AddRemoveElementPage extends Page {
+  get addElement() {
+    return $('//button[contains(text(),"Add Element")]');
+  }
+
+  get deleteElement() {
+    return $("button.added-manually");
+  }
+
+  async clickAddElement() {
+    await this.addElement.click();
+  }
+  async clickDeleteElement() {
+    await this.deleteElement.click();
+  }
+  async isElementDeleted() {
+    return await this.deleteElement.isExisting();

```

Comment:
If you have return you don't need await, also methods above should end with return too.